framework/wazuh/manager.py

```diff
@@ -439,6 +436,7 @@ def validation():
 
 
 def _parse_execd_output(output):
+    re_logtest = re.compile(r"^.*(?:ERROR: |CRITICAL: )(?:\[.*\] )?(.*)$")
```

Comment:
I agree the change of this regex, but don't with the new scope of the variable `re_logtest`. I think it's better to compile the regex only once (when the module is imported). However, since the variable is intended to be used only at module scope, I would rename it to `_re_logtest`.